### ISSUE TITLE
IPV4_DST_ADDR column is randomized at the beginning of the preprocessing

### ISSUE THREAD

**Issue by lorenzo9uerra** [renj-xu/NEGSC#6]
The column of the dataframe "IPV4_DST_ADDR" is randomized with IP addresses inside the range `[0xac100001,0xac1f0001]` which seems pretty arbitrary. This is causing the network graph to be manipulated. In this way, many network flows that would normally connect two nodes, will connect the source node with a random destination node instead, even before the train/test split. All information about multiple flows between the same two nodes is lost.

Why are you doing this, since there is no mention of it in the paper?
This could completely invalidate the results.

**Comment by ku1gga** [comment]
I am a noob, but I recommend ​you to read the paper 'E-GraphSAGE: A Graph Neural Network-Based Intrusion Detection System for IoT.'; ​You can find ​the answer there.

**Comment by lorenzo9uerra** [comment]
Thank you for the reply! I did read the E-GraphSAGE paper, however, in that paper a supervised learning approach is used and they randomize the **source** IP addresses, while in this case the **destination** IP address is randomized, which defies the same logic. To give context, the E-GraphSAGE paper reads: 

> To construct the network graph from the flow data, we mapped the original source IP addresses to randomly assigned IP addresses in the range from 172.16.0.1 to 172.31.0.1. The reason for this is the fact that in a lot of the NIDS datasets used in this paper, only a small number of IP addresses were used as the source of the attacks. The random mapping avoids the potential problem of the source IP addresses providing an unintentional label for attack traffic.
